OpenSSL-Strategy.md

```diff
@@ -107,6 +107,28 @@ For Node.js >= 10.16.0:
   supported by default, only by explicit run-time configuration.
 * FIPS: not supported
 
+XXX
+
+https://ci.nodejs.org/job/node-test-commit-linux-containered/11905/
+https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu1604_sharedlibs_openssl102_x64/11899/consoleText
+
+makes it seem 1.0.2 is supported, but I can't reproduce that build.
```

Comment:
@sam-github https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu1604_sharedlibs_openssl102_x64/11899/ is from a Node.js 6 build:

![image](https://user-images.githubusercontent.com/5445507/55421849-2496b500-5572-11e9-97a2-5bad2920ae97.png)


https://ci.nodejs.org/job/node-test-commit-linux-containered/11905/ skipped building against openssl 1.0.2, openssl 1.1.1 and fips.

Reply:
@richardlau So, when I look at https://ci.nodejs.org/job/node-test-commit-linux-containered/11905/, and its a v10.x build, all the light green checkmarks at the bottom are platforms that were skipped? Wow. And thanks!

On the plus side, that confirms that I'm not totally insane, and that openssl 1.0.2 is not in fact supported by node 10.x.

Reply:
> 
> 
> @richardlau So, when I look at https://ci.nodejs.org/job/node-test-commit-linux-containered/11905/, and its a v10.x build, all the light green checkmarks at the bottom are platforms that were skipped? Wow. And thanks!
> 

Correct. The faded light green is skipped.